velox/common/caching/SsdFileTracker.h

```diff
@@ -34,7 +34,9 @@ class SsdFileTracker {
   }
 
   void regionRead(int32_t region, int32_t bytes) {
-    regionScores_[region] += bytes;
+    // Cap region scores to UINT64_MAX to avoid overflow.
+    regionScores_[region] +=
```

Comment:
How easy is it for us to use `double` for `regionScores_`?

Reply:
Yeah, we could use double here and make sure checkpoint works properly. Either way is fine.
```
if (regionScores_[region] >= UINT64_MAX - bytes) {
   regionScores_[region] = UINT64_MAX;
} else {
   regionScores_[region] += bytes;
}
```